stellargraph/mapper/full_batch_generators.py

```diff
@@ -175,6 +175,9 @@ def flow(self, node_ids, targets=None):
             and :meth:`predict`
 
         """
+        if (self.multiplicity == 2) and isinstance(node_ids, tuple):
+            node_ids = np.stack(node_ids[:2], axis=1)
```

Comment:
Hmm I think we could move this to `FullBatchLinkGenerator.flow` instead? Then we can improve the docstrings together:
```python
class FullBatchLinkGenerator(FullBatchGenerator):
    ...
    def flow(link_ids, targets=None):
        """
            link_ids: an EdgeList or an iterable of link ids specified as tuples of node ids
        """
        if isinstance(link_ids, tuple):
            link_ids = np.stack(link_ids[:2], axis=1)
        ...

```

There's also a line in the docstring for `FullBatchLinkGenerator` that should be updated:
```
Use the :meth:`flow` method supplying the links as a list of (src, dst) tuples
```